onnx/backend/test/runner/__init__.py

```diff
@@ -169,15 +169,18 @@ def tests(self):  # type: () -> Type[unittest.TestCase]
     def assert_similar_outputs(cls, ref_outputs, outputs, rtol, atol):  # type: (Sequence[Any], Sequence[Any], float, float) -> None
         np.testing.assert_equal(len(outputs), len(ref_outputs))
         for i in range(len(outputs)):
-            np.testing.assert_equal(outputs[i].dtype, ref_outputs[i].dtype)
-            if ref_outputs[i].dtype == np.object:
+            if isinstance(outputs[i], (list, tuple)):
```

Comment:
Just out of curiosity: is there any example for tuple output?

Reply:
Acutally I did not hit tuple output so far. Just add tuple in case for some special case. Let me know I should keep it or remove it.